python/pfs/drp/stella/estimateRadialVelocity.py

```diff
@@ -143,6 +144,49 @@ def run(self, spectrum: PfsFiberArray, modelSpectrum: PfsSimpleSpectrum) -> Stru
             # We have to think of the error...
             return Struct(velocity=velocity, error=np.nan, fail=fail, crossCorr=crossCorr)
 
+        if self.config.findMethod == "quadratic":
```

Comment:
I suggest moving the code under the {{if}} blocks into their own methods. This will make the code easier to read, and allow subclassing to change behaviour later.